src/app/backend/resource/node/list.go

```diff
@@ -100,11 +104,17 @@ func toNodeList(nodes []api.Node, dsQuery *dataselect.DataSelectQuery, heapsterC
 	return nodeList
 }
 
-func toNode(node api.Node) Node {
+func toNode(node api.Node, pods *api.PodList) Node {
+	allocatedResources, err := getNodeAllocatedResources(node, pods)
+	if err != nil {
+		log.Printf("Couldn't get allocated resources of %s node\n", node.Name)
```

Comment:
It uses the same format as `kubectl describe node ...` and very similar logic behind it:

![image](https://cloud.githubusercontent.com/assets/2823399/26572614/ea146864-451a-11e7-88e1-affe35e3ba42.png)

If cluster won't be able to return data and it will throw an error our backend will return `nil` to the frontend. Otherwise, if data is here, we should not alter it in any way I think. 

Still few checks in frontend part are missing.